src/lib/data/near.js

```diff
@@ -184,6 +254,49 @@ const checkFakKey = (rpcResponse, contract, methodNames) => {
   return true;
 }
 
+async function submitFakCalimeroTransaction(componentName, near, contractName, methodName, args, gas, deposit) {
+  const key = localStorage.getItem(
+    await getCalimeroFakKey(componentName, near, contractName)
+  );
+
+  if(!key) {
+    throw new Error(
+      "Method: Near.fakCalimeroCall. Requires requestAccessKey to be called first"
+    );
+  }
+
+  const accountId = await getCurrentAccount(near);
+  const keyStore = new nearAPI.keyStores.InMemoryKeyStore();
+  const keyPair = nearAPI.KeyPair.fromString(key);
+  keyStore.setKey(CalimeroConfig.networkId, accountId, keyPair);
+
+  const calimeroConnection = await nearAPI.connect(
```

Comment:
Can we reuse the Calimero connections in the near object?

Reply:
Used existing Calimero connection with updated keystore and signer, tested and works.